code/mob/living/critter/small_animal.dm

```diff
@@ -84,13 +90,15 @@ ABSTRACT_TYPE(/mob/living/critter/small_animal)
 			src.is_pet = (copytext(src.name, 1, 2) in uppercase_letters)
 		if(in_centcom(loc) || current_state >= GAME_STATE_PLAYING)
 			src.is_pet = 0
-		if(src.is_pet)
+		if(src.is_pet || src.player_can_spawn_with_pet) // Also track player pets
```

Comment:
Probably best to only track it if it *is* a pet, not if this type can spawn as one?

Reply:
Mentioned this as well, what is considered a "pet" by the game is very arbitrary. It tracks random things like seals, but also command pets.